sdk/maps/Azure.Maps.Search/CHANGELOG.md

```diff
@@ -0,0 +1,112 @@
+# Release History
+
+## 1.0.3-beta.20 (2022-04-26)
```

Comment:
Not sure why we have theses versions, but these versions should be removed and change to 1.0.0-beta.1, since we never release it yet.